dotnet/README.md

```diff
@@ -77,7 +77,7 @@ requirements and setup instructions.
 3. [Running AI prompts from file](./notebooks/02-running-prompts-from-file.ipynb)
 4. [Creating Semantic Functions at runtime (i.e. inline functions)](./notebooks/03-semantic-function-inline.ipynb)
 5. [Using Kernel Arguments to Build a Chat Experience](./notebooks/04-kernel-arguments-chat.ipynb)
-6. [Creating and Executing Plans](./notebooks/05-using-the-planner.ipynb)
+6. [Introduction to the Planning/Function Calling](./notebooks/05-using-function-calling.ipynb)
```

Comment:
Just `Introduction to Function Calling`, I don't think we need to keep the reference to planning